### PR TITLE
Constants

### PR DESCRIPTION
**Pull request by john-breton**
- Extracted some Strings out to be constants in the NetworkManagementClient class.
- Reverted the change in getNetworkAlarms() in the NetworkManagementSystem so that a String is returned (and also made sure the returned String is printed in both of the clients). 
- Fixed some weird indentation in the NetworkManagementClient class.
- Added an else condition to listRuByParam() that will inform the user if an invalid parameter made it to the method.

### CODE REVIEW

File: src/main/java/mediator/Mediator.java [hunk 1]
```diff
@@ -161,7 +161,7 @@ private synchronized Carrier createCarrier(List<RfPort> rfPorts, FrequencyBand c
      * List any registered radio units that contain the parameter being passed.
      *
      * @param param The parameter that will be used to query for radio units. Examples
-     *              include a RatType, a RadioUnitState, a FrequencyBand, or a String name.
+     *              include a RatType, a RadioUnitStateE, a FrequencyBand, or a String name.
```
Comment (matthewkwsiu) [inline src/main/java/mediator/Mediator.java:164]: To clarify, what was RadioUnitStateE? Is it the error?

**Review by matthewkwsiu — APPROVED** [review]
Looks good to me. Thanks for adding the constants

File: src/main/java/mediator/Mediator.java (hunk 1 above)
Comment (john-breton) [reply src/main/java/mediator/Mediator.java:164]: I believe it is the enum that contains the different RadioUnitStates.

Comment (matthewkwsiu) [reply src/main/java/mediator/Mediator.java:164]: Ah, that makes sense.

**Pull request merged**